sympy/simplify/hyperexpand.py

```diff
@@ -2304,7 +2306,7 @@ def do_slater(an, bm, ap, bq, z, zfinal):
                 # Now sum the finitely many residues:
                 # XXX This speeds up some cases - is it a good idea?
                 integrand = expand_func(integrand)
-                for r in range(lu):
+                for r in range(int(lu)):
```

Comment:
So in that case I'm not sure. `int` will always round down but it sounds like `lu` will be near an integer, but not necessarily greater than the integer it is near. Would `round(lu)` be better?